src/base/static/components/organisms/map-measurement-overlay.tsx

```diff
@@ -0,0 +1,408 @@
+/** @jsx jsx */
+import * as React from "react";
+import { css, jsx } from "@emotion/core";
+import styled from "@emotion/styled";
+import PropTypes from "prop-types";
+import { featureCollection, point, lineString, polygon } from "@turf/helpers";
+import area from "@turf/area";
+import distance from "@turf/distance";
+import { geoPath, geoTransform } from "d3-geo";
+import WebMercatorViewport from "viewport-mercator-project";
```

Comment:
If types definitions aren't included in the `viewport-mercator-project` package, then we should install this package: https://www.npmjs.com/package/@types/viewport-mercator-project

which will include the following types: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/viewport-mercator-project/index.d.ts